website/pages/docs/guides/variants.mdx

```diff
@@ -130,6 +130,16 @@ final style = Style(
 );
 ```
 
+## Prioritizing Variants
```

Comment:
Prioritization should be only on `ContextVariants` as the use of `Variants` are explicit in its priority.

Reply:
So should I remove the priority parameter in the Variants constructor?

Reply:
I don't know if I'm sure about this. Imagine that we have a variant, such as disabled in a button. It's a normal variant that needs priority if compared with other ones.